app/views/migration/index.html.mustache

```diff
@@ -1,3 +1,3 @@
 {{>atoms/meta/_head}}
-{{>templates/Search/Search-static-page}}
+{{>templates/Search/Search-stories-index}}
```

Comment:
Could we create a new template directory named "Stories" and move these new templates there?

Reply:
I've relocated and renamed the relevant pages to:
  templates/Stories/Stories-Index
  templates/Stories/Stories-New